src/common/fs_file.h

```diff
@@ -2,26 +2,27 @@
 
 #include <bit>
 #include <cstdio>
-#include <string>
 #include <span>
+#include <string>
 #include <vector>
 
 #include "common/types.h"
 
 namespace Common::FS {
 
-enum class OpenMode : u32 {
-    Read = 0x1,
-    Write = 0x2,
-    ReadWrite = Read | Write
-};
+enum class OpenMode : u32 { Read = 0x1, Write = 0x2, ReadWrite = Read | Write };
```

Comment:
Enums should not be single line

Reply:
this i causing from clang formating , maybe we need a clang format ci ? or improving the existing clang file?